packages/paste-website/src/pages/components/meter/index.mdx

```diff
@@ -0,0 +1,233 @@
+
+<LivePreview scope={{Meter, MeterLabel, HelpText, useUID, Box}} language="jsx" noInline>
+  {minMaxMeter}
+</LivePreview>
+
+### Hidden value label
+
+<LivePreview scope={{Meter, useUID, Box}} language="jsx" noInline>
+  {hiddenValueLabel}
+</LivePreview>
+
+## Composition notes
+
+The Meter label should communicate what the Meter is measuring. Where possible, avoid a label that wraps onto two lines.
+
+A Meter almost always will include a numerical value, the value label. When using the `valueLabel` prop, consider what type of value would be most useful for a user to see (for example, “50%” vs. “5000 of 10,000”).
+
+Use Help Text to offer additional information to contextualize or help the user understand the Meter.
+
+## When to use a Meter
+
+<DoDont>
+  <Do
+    title="Do"
+    body="Use a meter when a user needs to understand a measurement within a range (like how much data has been used)."
+    center
+  >
+    <Box width="80%">
+      <MeterLabel htmlFor="do1">Data usage</MeterLabel>
+      <Meter id="do1" value={79} />
+    </Box>
+  </Do>
+  <Dont
+    title="Don't"
+    body="Don't use a Meter to represent a path with a completion point (like uploading a file or filling out a form)."
+    center
+  >
+    <Box width="80%">
+      <MeterLabel htmlFor="dont1">File upload status</MeterLabel>
+      <Meter id="dont1" value={23} />
+    </Box>
+  </Dont>
+</DoDont>
+
+<DoDont>
+  <Do
+    title="Do"
+    body="When using a custom value label, use only the format that is the most useful for a user to know."
+    center
+  >
+    <Box width="80%">
+      <MeterLabel htmlFor="do2" valueLabel="$1,259">
+        Balance
+      </MeterLabel>
+      <Meter
+        id="do2"
+        aria-describedby="help"
+        value={1259}
+        minValue={0}
+        maxValue={1500}
+        minLabel="$0"
+        maxLabel="$1,500"
+      />
+      <HelpText id="help">Complete balance due at the end of the billing cycle.</HelpText>
+    </Box>
+  </Do>
+  <Dont title="Don't" body="Don't show multiple formats of the same value when possible." center>
+    <Box width="80%">
+      <MeterLabel htmlFor="dont2" valueLabel="$1,259 of $1,500 (84%)">
+        Balance
+      </MeterLabel>
+      <Meter
+        id="dont2"
+        aria-describedby="help2"
+        value={1259}
+        minValue={0}
+        maxValue={1500}
+        minLabel="$0"
+        maxLabel="$1,500"
+      />
+      <HelpText id="help2">Complete balance of $1,500 due at the end of the billing cycle.</HelpText>
+    </Box>
+  </Dont>
+</DoDont>
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/core - or - yarn add @twilio-paste/meter
+```
+
+#### Usage
+
+```jsx
+import {Meter, MeterLabel} from '@twilio-paste/core/meter';
```

Comment:
suggestion: let's add the other imports like HelpText and useUID

Reply:
good callout ty